doc/user_guide/axes.rst

```diff
@@ -107,6 +107,17 @@ attributes, e.g.:
     >>> s.axes_manager[0]
     <X axis, size: 20, index: 0>
 
+.. versionadded:: 2.2
+    `set` method for `navigation_axes` and `signal_axes`.
+
+It is also possible to set multiple attributes of multiple axes at once, using the :meth:`~.misc.utils.TupleSA.set`
```

Comment:
`~.misc.utils.TupleSA.set` needs to be added to the API reference.

Reply:
Done